Tests/Binary/Loader/GridFSLoaderTest.php

```diff
@@ -23,6 +23,10 @@ class GridFSLoaderTest extends \PHPUnit_Framework_TestCase
 
     public function setUp()
     {
+        if (!class_exists('Doctrine\ODM\MongoDB\DocumentRepository')) {
+            $this->markTestSkipped('Doctrine ODM not installed');
```

Comment:
in case our tests require the mongo pecl ext as well to run, we should also check for their existence to determine if to skip